src/ngraph/frontend/onnx_import/op/unsqueeze.cpp

```diff
@@ -0,0 +1,66 @@
+//*****************************************************************************
+// Copyright 2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include <numeric>
+
+#include "unsqueeze.hpp"
+
+#include "ngraph/op/reshape.hpp"
+
+#include "exceptions.hpp"
+#include "utils/reshape.hpp"
+
+namespace ngraph
+{
+    namespace onnx_import
+    {
+        namespace op
+        {
+            NodeVector unsqueeze(const Node& node)
+            {
+                NodeVector inputs{node.get_ng_inputs()};
+                auto data = inputs.at(0);
+                auto data_shape = data->get_shape();
+                auto axes = node.get_attribute_value<std::vector<int64_t>>("axes");
+                if (axes.empty())
+                {
+                    throw error::parameter::Value(
+                        "Unsqueeze", node.get_name(), "axes attribute is mandatory.");
+                }
+                std::sort(axes.begin(), axes.end(), std::greater<int64_t>());
+
+                // Generate an increasing sequence (0,1,2,3..) as input_order for Reshape
+                ngraph::AxisVector input_order(data_shape.size());
+                std::iota(std::begin(input_order), std::end(input_order), 0);
+
+                for (auto axis : axes)
+                {
+                    if (axis < 0 || axis > data_shape.size())
+                    {
+                        throw error::parameter::Value(
```

Comment:
Can we skip invalid attribute (just reporting warning) and continue with other attributes (robustness principle), or invalid attribute means we can not continue as it is now?